.github/workflows/develop-push-deploy.yml

```diff
@@ -47,26 +47,35 @@ jobs:
         run: docker login -u addis -p ${{ secrets.dockerRegistryPassword }}
 
       - name: build docker image
-        run: ./build-docker-with-tag.sh GOOGLE Test https://mcda-test.drugis.org test
+        run: ./build-docker.sh GOOGLE Test https://mcda-test.drugis.org test
 
       - name: docker push
         run: docker push addis/mcda:test
 
       - name: pull latest from drugis on the server
         run: ssh -o StrictHostKeyChecking=no deploy@mcda-test.drugis.org 'cd mcda-elicitation-web && git checkout develop && git pull'
-      # StrictHostKeyChecking=no ensures no prompt for 'add to known hosts'
+
+      - name: pull image on server
+        run: ssh -o StrictHostKeyChecking=no deploy@mcda-test.drugis.org 'docker pull addis/mcda:test'
 
       - name: stop MCDA docker container on the server
-        run: ssh -o StrictHostKeyChecking=no deploy@mcda-test.drugis.org 'docker stop mcda'
+        run: ssh -o StrictHostKeyChecking=no deploy@mcda-test.drugis.org 'docker stop mcda || true'
 
-      - name: update database on the server
-        run: ssh -o StrictHostKeyChecking=no deploy@mcda-test.drugis.org 'mcda-elicitation-web/liquibase/update-db.sh update'
+      - name: update database
+        run: |
+          cd liquibase
+          cp ../database.pg.sql liquibase-changelog.sql
+          docker run --rm -v $(pwd):/liquibase/ --network host -e "LIQUIBASE_URL=jdbc:postgresql://psql-test.drugis.org:5432/mcdaweb" -e "LIQUIBASE_USERNAME=mcdaweb" -e "LIQUIBASE_PASSWORD=${{ secrets.mcdaTestDbPassword }}" -e "LIQUIBASE_CHANGELOG=liquibase-changelog.sql" webdevops/liquibase:postgres update
```

Comment:
is it possible to put the linebreaks back in somehow? This is a lot harder to parse than the `\`-separated one